explorer/src/components/NavigationList/components/ListItemWithSubmenu/index.tsx

```diff
@@ -12,16 +12,16 @@ interface ListItemWithSubmenuProps {
 export const ListItemWithSubmenu: React.FC<ListItemWithSubmenuProps> = ({ name, submenu }) => {
   const screen = useWindowDimensions();
   const [showSubmenu, setShowSubmenu] = useState<boolean>(false);
-  const isAdaptive = screen.sm || screen.md;
+  const isAdaptive = !screen.xl;
 
   return (
     <div
       onMouseEnter={() => !isAdaptive && setShowSubmenu(true)}
       onMouseLeave={() => !isAdaptive && setShowSubmenu(false)}
```

Comment:
I'm not sure that it going to work correctly on touchscreen devices with `screen.xl` resolution